### ISSUE TITLE
How can I improve the effect of my model

### ISSUE THREAD

**Issue by kkeleve** [facebookresearch/fairseq#4075]
I am currently training a Chinese to English transformer base model，the casia2015 in WMT used in the data set has a total of about 100w sentences。I think the bleu I trained is not the best，I want to know if it is a super parameter that needs to be changed。I will show my training log。

In data preparation, I cleaned the data with moses script, tokenized words, and then applied BPE using subword-nmt, where I set number of BPE tokens to 32000
For preprocessing:
 I also used pkuseg (better than jieba)  for Chinese, and I used moses for English. After that I applied BPE
fairseq-preprocess --source-lang ${src} --target-lang ${tgt} \
    --trainpref ${data_dir}/train --validpref ${data_dir}/valid --testpref ${data_dir}/test \
    --destdir ${data_dir}/data-bin

For training:
CUDA_VISIBLE_DEVICES=0 nohup fairseq-train ${data_dir}/data-bin \
  -a transformer --optimizer adam --source-lang ${src} --target-lang ${tgt} \
  --label-smoothing 0.1 --dropout 0.3 --max-tokens 4000 \
  --lr-scheduler inverse_sqrt --weight-decay 0.0001 \
  --criterion label_smoothed_cross_entropy --max-update 200000 \
  --warmup-updates 10000 --warmup-init-lr '1e-7' --lr '0.001' \
  --adam-betas '(0.9, 0.98)' --adam-eps '1e-09' --clip-norm 25.0 \
  --update-freq 4 --max-epoch 25 \
  --tensorboard-logdir ~/nmt/log/tensorboardlog_tc4 \
  --keep-last-epochs 2 --save-dir ${model_dir}/checkpoints_tc4 > ~/nmt/log/train_tc4.log 2>&1 &
I trained this on a data set of ~1M samples, on 3090 GPU, for about 3.5 hours. The 21 bleu  is still some distance away from 24，I want to know why。。。

[train_tc4.log](https://github.com/pytorch/fairseq/files/7709018/train_tc4.log)

#### What's your environment?
 - fairseq Version (e.g., 1.0 or main):1.0
 - PyTorch Version (e.g., 1.0) 1.10
 - OS (e.g., Linux): ubuntu18
 - How you installed fairseq (`pip`, source):
 - Build command you used (if compiling from source):
 - Python version:3.8
 - CUDA/cuDNN version:11.3
 - GPU models and configuration:3090*1
 - Any other relevant information:
